app/src/main/java/com/mertceyhan/bitcoinmarket/features/market/ui/MarketScreen.kt

```diff
@@ -115,6 +67,64 @@ fun MarketScreen(marketViewModel: MarketViewModel = hiltViewModel()) {
     }
 }
 
+@Composable
+fun ScreenContent(scrollState: ScrollState, viewState: MarketScreenViewState? = null, marketViewModel: MarketViewModel, isShowShimmer: Boolean) {
```

Comment:
I'm not sure passing null `viewState` to UI 🤔 I think the UI shouldn’t care about the nullability. Do you have a better solution or thought for here?